chsdi/lib/helpers.py

```diff
@@ -378,3 +378,9 @@ def get_loaderjs_url(request):
 def decompress_gzipped_string(string):
     content = gzip.GzipFile(fileobj=StringIO.StringIO(string))
     return content.read()
+
+
+def order_mapsheet(lang):
+    shopUrl = 'https://beta.shop.swisstopo.ch'
```

Comment:
meaning use `request.registry.settings` and add a variable in `production.ini.in`.